wger/core/templates/template.html

```diff
@@ -100,8 +100,10 @@
     <div class="container">
         <div class="row">
             <div class="col-md-9" style="padding-bottom: 0.5em;">
-                <span class="name">wger</span>
-                <span class="regular">| Workout Manager</span>
+                <a href="/" style="text-decoration:none;">
```

Comment:
use the `url tags`   **{% url  ???%}** tags its much safer in case we decide to change the url pattern